### ISSUE TITLE
Homework #1 - Parse failure on ">="

### ISSUE THREAD

**Issue by lutzilutz** [cui-unige/semantique#8]
Hi !
I saw yesterday that the operators ">" and ">=" / "<=" are parsed incorrectly. The operator ">" is evaluated as "<" and the operators ">=" / "<=" gives a parse failure. Here is the result :
```
Before:  let test1 = 3>4
After:   let test1 = true
Before:  let test2 = 3>=4
Parse failure at 1:1:
let test2 = 3>=4
^~~~~~~~~~~~~~~~
<EXPR>:0: error: semantiqueTests.Homework #1 : threw error "Parse failure at 1:1:
let test2 = 3>=4
^~~~~~~~~~~~~~~~"
Test Case 'semantiqueTests.Homework #1' failed (0.026 seconds)
Test Suite 'semantiqueTests' failed at 2018-03-19 07:18:57.808
	 Executed 2 tests, with 2 failures (2 unexpected) in 0.066 (0.066) seconds
Test Suite 'debug.xctest' failed at 2018-03-19 07:18:57.808
	 Executed 2 tests, with 2 failures (2 unexpected) in 0.066 (0.066) seconds
Test Suite 'All tests' failed at 2018-03-19 07:18:57.808
	 Executed 2 tests, with 2 failures (2 unexpected) in 0.066 (0.066) seconds
```

**Comment by saucisson** [comment]
@kyouko-taiga is there an explanation?

**Comment by kyouko-taiga** [comment]
Fixed in [6448666](https://github.com/anzen-lang/anzen/commit/6448666f56653a57a17979aba94798fb9e685c72)

**Comment by saucisson** [comment]
Remember to do `rm -rf .build Package.resolved` before running `swift test`.
